modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-filtering-ebpf-rule_{context}"]
+= Filtering eBPF flow data using a global rule
+You can configure the `FlowCollector` to filter eBPF flows using a global rule as a method to control the flow of packets cached in eBPF's flows table based on certain configuration.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for *Network Observability*, select *Flow Collector*.
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource. A few sample configurations are as follows:
++
+
+[%collapsible]
+.Filter Kubernetes service traffic to specific POD IP endpoint
+====
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: Direct
+  agent:
+    type: eBPF
+    ebpf:
+      flowFilter:
+        action: Accept  <1>
+        cidr: 172.210.150.1/24 <2>
+        protocol: SCTP
+        direction: Ingress
+        destPortRange: 80-100
+        peerIP: 10.10.10.10
+        enable: true    <3>
+----
+<1> The required `action` parameter describes the action to be taken for the flow filter rule. Possible values are `Accept` or `Reject`. 
+<2> The required `cidr` parameter provides the IP address and CIDR mask for the flow filter rule, and supports IPv4 and IPv6 address formats. If you want to match against any IP address, you can use `0.0.0.0/0` or `::/0` for IPv4 and IPv6, respectively.
+<3> You must set `spec.agent.ebpf.flowFilter.enable` to `true` to enable the feature.
+====
++
+[%collapsible]
+.See flows to any addresses outside the cluster 
+====
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: Direct
+  agent:
+    type: eBPF
+    ebpf:        
+      flowFilter:
+        action: Accept  <1>
+        cidr: 0.0.0.0/0 <2>
+        protocol: TCP
+        direction: Egress
+        sourcePort: 100
+        peerIP: 192.168.127.12 <3>
+        enable: true    <4>
+----       
+<1> You can `Accept` flows based on the criteria in the `flowFilter` specification.
+<2> The `cidr` value of `0.0.0.0/0` matches against any IP address.
+<3> See flows after `peerIP` is configured with `192.168.127.12`. 
```

Comment:
```suggestion
<3> The flows after `peerIP` are configured with `192.168.127.12`. 
```
This sentence didn't make sense to me, but I am, admittedly, not familiar with the product. Do my edits change the meaning of the footnote?

Reply:
I think this is meant more like, You can see flows once the peerIP is configured